apps/trading-e2e/src/support/step_definitions/markets-page.step.js

```diff
@@ -0,0 +1,21 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+import MarketsPage from '../pages/markets-page';
+const marketsPage = new MarketsPage();
+
+When('I click on market for {string}', (marketText) => {
+  marketsPage.clickOnMarket(marketText);
+});
+
+When('I click on active market', () => {
+  if (Cypress.env('bypassPlacingOrders' != true)) {
+    marketsPage.clickOnMarket('Active');
+  } else marketsPage.clickOnTopMarketRow();
```

Comment:
formatting.

Can you add the following to the eslint config please:

```
      "curly": "warn",
```

Reply:
Done